model_utils/fields.py

```diff
@@ -126,6 +126,33 @@ def deconstruct(self):
         return name, path, args, kwargs
 
 
+class PersistentMonitorField(MonitorField):
+    """
+    Like MonitorField, but once the timestamp for the first change is recorded
+    it is never set again
+    """
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('default', now())
```

Comment:
__init__ runs when the model is instantiated, so I don't think that will be an issue. It could be set to None if that's more preferable...